packages/wasm/src/wasm_package.rs

```diff
@@ -92,11 +128,21 @@ impl WrapPackage for WasmPackage {
     }
 
     fn create_wrapper(&self) -> Result<Arc<dyn Wrapper>, polywrap_core::error::Error> {
-        let wasm_module = self.get_wasm_module()?;
+        let wasm_bytes = self.get_wasm_module()?;
+
+        let mut wasm_module = self.wasm_module.lock().unwrap();
+
+        let compiled_module = if wasm_module.is_some() {
+            wasm_module.clone().unwrap().compile()?
```

Comment:
Is this clone copying the whole buffer, or just creating another mutex, which references the same underlying buffer?

Reply:
For the CompiledWasmModule (wasmer::Module) it is a shallow copy.
For other variants of WasmModule it is just copying an Arc<[u8]> now (recent commit)

Reply:
Awesome :)